src/.clang-format

```diff
@@ -25,53 +25,74 @@ IndentWidth: 4
 UseTab: Never
 TabWidth: 8
 AccessModifierOffset: -4
-ConstructorInitializerIndentWidth: 4
-ContinuationIndentWidth: 4
+ConstructorInitializerIndentWidth: 0
+ContinuationIndentWidth: 8
 IndentCaseLabels: false
 IndentWrappedFunctionNames: false
 NamespaceIndentation: None
 
 # line breaks
-AlwaysBreakAfterDefinitionReturnType: false
+AlwaysBreakAfterDefinitionReturnType: None
+AlwaysBreakAfterReturnType: None
 AlwaysBreakBeforeMultilineStrings: false
 AlwaysBreakTemplateDeclarations: true
-BreakBeforeBinaryOperators: true
-BreakBeforeBraces: Linux
+BraceWrapping:
+  AfterClass: false
+  AfterControlStatement: false
```

Comment:
I believe this to be so that the '{' for an `if (...) {` is moved to the next line if true - so I am happy with false here.